### ISSUE TITLE
No Contract Date in AMC 

### ISSUE THREAD

**Issue by sabinsp09** [azharakbar/lafi-sa.issues#13]
**Is your feature request related to a problem? Please describe.**
There is no Date of Issue of Contract can be seen in the Annual Maintenance Contract Print. So please include this option under the Contract Number.

**Describe the solution you'd like**
The Date of Issue will be the date when contracted is drafted and finalized to print.
![image](https://user-images.githubusercontent.com/61094152/76738163-2b1a3e80-677b-11ea-9a97-7ee630cf726f.png)

**Comment by sabinsp09** [comment]
@azharakbar dear...i can not give "New Ticket" label. Labels are off for me.

**Comment by azharakbar** [comment]
@sabinsp09 
I have added the labels

Pls contact @majeedpalakkal and tell him to verify the tickets that are open

I can proceed only after that

**Comment by azharakbar** [comment]
@majeedpalakkal 
Pls veify the requirement and propose the position of date in the AMC print format

**Comment by azharakbar** [comment]
Duplicate of #17
